community/contributing/content_guidelines.rst

```diff
@@ -0,0 +1,95 @@
+.. _doc_content_guidelines:
+
+Content guidelines
+==================
+
+This document is here to help us assess what we should include in the official
+documentation. Below, you will find a couple of principles and recommendations
+to write accessible content.
+
+We want to achieve two goals:
+
+1. **Empathize with our users**. We should write in a way that makes it easy for
+   them to learn from the docs.
+2. **Write a complete reference manual**. Our goal here is not to teach
+   programming foundations. Instead, we should provide a reference for how
+   Godot’s features work.
```

Comment:
I think the de facto convention for apostrophes has been to use `'` instead of the typographical variant that most keyboard layouts can't input `’`.

I did remove some occurrences of `’` in the past, and currently there doesn't seem to be any in the repo.